extensions/extension_types.yaml

```diff
@@ -8,3 +8,6 @@ types:
     structure:
       start: point
       end: point
+  - name: geometry
```

Comment:
So this represents a series of connected points.  Is there a structure that represents holes such as a donut?

Reply:
I don't think there is a singular structure that can represent a donut, but you could have 2 sets of connected points that each form a circle and one could be entirely within the other.  Does that answer your question?

Reply:
I found ST_MakePolygon which can represent a polygon within a geometry which indeed is constructed from 2 sets of connected points.  It looks like everything is defined as a geometry type in POSTGIS.